src/line.cpp

```diff
@@ -81,12 +79,13 @@ nzl::Line::LineImp::LineImp() : program{make_program()} {
   glDisableVertexAttribArray(0);
   glBindBuffer(GL_ARRAY_BUFFER, 0);
   glBindVertexArray(0);
-}
 
-nzl::Line::LineImp::~LineImp() noexcept {
-  /// @TODO: Add error checking!
+  nzl::check_gl_errors();
+}
+nzl::Line::LineImp::~LineImp() {
```

Comment:
@arrieta Is it okay to have a destructor which can throw an exception in this case?